app/scripts/cookie-consent.js

```diff
@@ -0,0 +1,4 @@
+// NHS.UK cookie consent v0.5.2
+// This file has been modified for the NHS digital service manual
```

Comment:
This script has been modified for the NHS digital service manual, we're probably best off using the original one from the cookie consent repo: https://github.com/nhsuk/cookie-consent/archive/0.5.1.zip

Reply:
Also this file isn't actually being initialised

`cookie-consent.min.js:1 Failed to load resource: the server responded with a status of 404 (Not Found)`

Will need renaming to `cookie-consent.min.js`

Reply:
@AdamChrimes I am a bit confused by this I think I was expecting the JS to compile from the checked-out cookie repo and add to my scripts folder a min version. That didn't happen so either I miss understood the repo or there is an error in JS build, which is probable. I don't suppose you have any suggestions.

Reply:
@BigEdDev I've fixed the application JavaScript and sorted the cookie logic in [0fe33b1](https://github.com/NHSDigital/governance/pull/3/commits/0fe33b1790e1e271153daf0c87e9800e6a94b764)